docs/src/SUMMARY.md

```diff
@@ -35,6 +35,7 @@
 - [The Snowcat Type Checker](./apalache/typechecker-snowcat.md)
 - [Supported Features](./apalache/features.md)
 - [Known Issues](./apalache/known-issues.md)
+- [Antipatterns](./apalache/antipatterns.md)
```

Comment:
Antipatterns is fairly well established terminology, afaik. It also emphasizes the "syntax to avoid" aspect. @konnov, @gabrielamafra, any preferences?

As for the indentation, I don't think it's wrong, the material goes beyond "Getting started", I suppose.